CSharpBashInterpreter/Commands/Abstractions/BaseCommandExecutable.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CSharpBashInterpreter.Utility;
+
+namespace CSharpBashInterpreter.Commands.Abstractions;
+
+/// <summary>
+///     Base class with boilerplate for command executables
+/// </summary>
+public abstract class BaseCommandExecutable : ICommandExecutable
+{
+    protected readonly string[] Tokens;
+
+    protected BaseCommandExecutable(IEnumerable<string> tokens)
+    {
+        Tokens = tokens.ToArray();
+    }
```

Comment:
```suggestion
    protected BaseCommandExecutable(IEnumerable<string> tokens)
        => Tokens = tokens.ToArray();
```
Так писать короче :)

Reply:
Fixed